index.html

```diff
@@ -7,6 +7,28 @@
     <title>Recipe App</title>
 </head>
 <body>
-    <h1>Recipe App</h1>
+    <section class="navigation">
+        <nav>
+            <input type="checkbox" id="check">
+            <label for="check" class="checkbtn">
+                <i class="fas fa-bars"></i>
+            </label>
+            <ul>
+                <li><a class="active" href="#">Home</a></li>
+                <li><a href="">Gallery</a></li>
+                <li><a href="">Contact Us</a></li>
+            </ul>
+        </nav>
+        <div class="container initial">
+            <h1 class="headerText">Recipe App</h1>
+            <form id="form">
+                <input name="recipeName" type="text" id="text" placeholder="Search Your Recipe..." value="">
+                <button>Search</button>
+            </form>
+            <div class="searchResult">
+                <ul class="recipes"></ul>
+            </div>
+        </div>
```

Comment:
This looks great. Just you missed section closing tag.

Reply:
Added closing tag for section.